pkg/packagekit/kpatch.jsx

```diff
@@ -0,0 +1,325 @@
+                            labelOff={_("Kernel live patching service is disabled")}
+                            onChange={c => this.setState({ futureEnabled: c })} />
+            </StackItem>
+            <StackItem>
+                <Switch isChecked={this.state.futureAuto}
+                            label={_("Subscribed to available kernel patches")}
+                            labelOff={_("Not subscribed to kernel patches")}
+                            onChange={c => this.setState({ futureAuto: c })} />
+            </StackItem>
+            <StackItem>
+                <List>{details.map(d => <ListItem key={d}>{d}</ListItem>)}</List>
+            </StackItem>
+        </Stack>;
+
+        return (<>
+            <div id="kpatch-settings">
+                <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                    <Flex grow={{ default: 'grow' }} alignItems={{ default: 'alignItemsBaseline' }}>
+                        <FlexItem>
+                            <b>{_("Kernel live patching")}</b>
+                        </FlexItem>
+                        <FlexItem>
+                            {state}
+                        </FlexItem>
+                    </Flex>
+                    <Flex>
+                        <Button variant="secondary"
+                                isSmall
+                                disabled={!this.props.privileged}
+                                onClick={action}>
+                            {actionText}
+                        </Button>
+                    </Flex>
+                </Flex>
+            </div>
+            <Modal position="top" variant="small" id="kpatch-setup" isOpen={this.state.showModal}
+                title={_("Kernel live patching settings")}
+                onClose={() => this.setState({ showModal: false })}
+                footer={
+                    <>
+                        <Button variant="primary"
+                                isLoading={ this.state.updating }
+                                isDisabled={ this.state.updating || details.length === 0 }
+                                onClick={ this.handleChange }>
+                            {_("Apply")}
+                        </Button>
+                        <Button variant="link"
+                                isDisabled={ this.state.updating }
+                                onClick={() => this.setState({ showModal: false })}>
+                            {_("Cancel")}
+                        </Button>
+                    </>
+                }>
+                {body}
+            </Modal>
+        </>);
+    }
+}
+
+KpatchSettings.propTypes = {
+    privileged: PropTypes.bool.isRequired,
+};
+
+export class KpatchStatus extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {
+            loaded: [],
+            installed: [],
+            changelog: null, // FIXME - load changelog
```

Comment:
Right, I forgot about this. I'll take a look at it now. But I guess even a follow up will do?

Reply:
Is that treated differently than a "normal" package changelog? But yes, fine for me to do that in a follow-up.

Reply:
no, it is the same changelog. Maybe a bit more parsing and pointing to CVEs...